openproject/client.py

```diff
@@ -70,101 +70,29 @@ class WorkPackages(SubClient):
         "custom_field_2": "customField2",
         "created_at": "createdAt",
         "updated_at": "updatedAt",
+        "lock_version": "lockVersion",
     }
 
+    def _api_payload_from_kwargs(self, **kwargs) -> WorkPackage:
+        data = {}
+        for arg, api_arg in self._args_api_mapping.items():
+            if arg in kwargs:
+                data[api_arg] = kwargs[arg]
```

Comment:
I suggest we use comprehension here, to LOCs and make it easier to debug with variables. Example:

```suggestion
        items = self._args_api_mapping.items():
        data = {api_arg: kwargs[arg] for arg, api_arg in items}
```

We need to verify this works though.

Reply:
Slightly different to handle `KeyError`

```python
items = self._args_api_mapping.items()
data = {api_args: kwargs[args] for args, api_args in items if args in kwargs}
```